lib/gather/gather-telemetry.js

```diff
@@ -56,7 +60,6 @@ module.exports = async function gatherTelemetry(url, puppeteerArgs = {}) {
         'Route',
         'Component',
         'Service',
-        'Helper',
```

Comment:
how are helpers identified?

Reply:
One thing I did learn is there is no Ember Type` Helper` like there is Component and Route.  The only thing I found the allows me to determine a helper is to use `isHelperFactory `property

Reply:
huh, interesting!